docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[rules-prebuilt]]
+== Install Elastic prebuilt rules
+
+:frontmatter-description: Start detections quickly with prebuilt rules designed and updated by Elastic.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
+
+* <<load-prebuilt-rules>>
+* <<prebuilt-rule-tags>>
+* <<select-all-prebuilt-rules>>
+* <<update-prebuilt-rules>>
+* <<download-prebuilt-rules>>
+* <<rule-prerequisites>>
+
+[NOTE]
+====
+* Apart from the Elastic Endpoint rule, prebuilt rules do not start running by default. You must first install the rules, then enable them.
+
+* You can't modify most settings on Elastic prebuilt rules. You can only edit <<rule-schedule, rule actions>> and <<add-exceptions, add exceptions>>. If you want to modify other settings on a prebuilt rule, you must first duplicate it, then make your changes to the duplicated rule. 
+====
+
+[float]
+[[load-prebuilt-rules]]
+=== Install and enable Elastic prebuilt rules
+
+. Go to *Manage* -> *Rules*. The badge next to *Add Elastic rules* shows the number of uninstalled prebuilt rules available. 
++
+[role="screenshot"]
+image::images/prebuilt-rules-add-badge.png[The Add Elastic Rules page]
```

Comment:
I know this is a long-ish image, but it might need to be a smidge larger. Idk if we can go over 100%, but if we can, here's something you can try:

```suggestion
image::images/prebuilt-rules-add-badge.png[The Add Elastic Rules page, 110%]
```

Reply:
Looks like you can't extend an image past the column's width (I tried both % and pixel count). The image looks readable on my monitor, but it's admittedly super high-def. I can try retaking a more compact image so the text is larger.

Reply:
Images updated: https://github.com/elastic/security-docs/pull/3552/commits/40d00a0c10ad864abf612b614a2224dcdb8aa47d